PR Details Modal: Svelte TS classes

estib-vega opened the pull request:
Use classes for the reactive logic for handling the PR title and body

Caleb-T-Owens commented on apps/desktop/src/lib/pr/prContent.svelte.ts:
```diff
@@ -1,114 +1,70 @@
-		if (params.isDisplay) return params.existingBody ?? '';
-		if (params.seriesDescription) return params.seriesDescription;
-		if (templateBody) return templateBody;
+	set(value: string) {
+		this.title = value;
+	}
```
ought this be `set current(value: string) {` ?

estib-vega replied:
We can do that, but I thought it would be nice to have a setter method that's explicit.

I have no strong feelings against it, though

Commit:
PR Details Modal: Svelte TS classes

Use classes for the reactive logic for handling the PR title and body
Suggestions - please squash this commit
